apps/chat/README.md

```diff
@@ -19,10 +19,13 @@ cd apps/chat
 npm run deploy
 ```
 
-3. Verify that the `src/backend/serverless/appconfig.json` is populated with the CloudFormation stack outputs. This JSON file used to
-   configure the app with `src/Config.js.`
+3. Run app locally
```

Comment:
this section is to serve the demo app via s3 + CF. there is another section below this to run it locally.

Reply:
s3 + CF currently doesn't work so hence combining the two now for temporary solution